src/commands/deploy.rs

```diff
@@ -200,11 +200,15 @@ impl DeployCommand {
 
     async fn compute_buildinfo(&self, cfg: &RawAppManifest) -> Result<BuildMetadata> {
         let mut sha256 = Sha256::new();
+        let mut app_folder = PathBuf::from(&self.app);
+        app_folder.pop();
 
         for x in cfg.components.iter() {
             match &x.source {
                 config::RawModuleSource::FileReference(p) => {
-                    let mut r = File::open(p)?;
+                    let mut absolute_path = PathBuf::from(&app_folder);
+                    absolute_path.push(p);
+                    let mut r = File::open(absolute_path)?;
```

Comment:
Please add a `with_context`, specifically to include the file name - that way the user gets a more informative error message if anything goes wrong - unfortunately the stdlib errors are a bit woeful for this.

Reply:
Updated it. Thanks for the suggestion! I am new to Rust and I was wondering how to show error messages with more content. I know how to do it now 👍